docker/optimizer-flink/Dockerfile

```diff
@@ -14,19 +14,23 @@
 #  See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG FLINK_VERSION=1.14
+ARG FLINK_VERSION=1.18.1-java8
```

Comment:
default flink_version in build.sh is 1.15.3, which uses java11. So ARG FLINK_VERSION=1.18.1 is needed?

Reply:
we should direct users to default to the latest supported version of Flink